packages/dev/docs/pages/releases/2022-05-27.mdx

```diff
@@ -0,0 +1,49 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: React 18 is now officially supported! Other updates include a major version bump to our workflow icons, support for emphasized Tabs, and various bug fixes to our existing components and hooks. Thank you to all who contributed to this release!
+date: 2022-05-27
+---
+
+# May 27, 2022 Release
+
+Ever since React 18 was announced last July, compatibility has been front of mind for the team. We are extremely pleased to announce that our library now officially supports React 18! Special thanks to [@eps1lon](https://github.com/eps1lon) for their
+help with getting our tests working with React 18.
+
+A major version bump to our workflow icons is included in this release, so be sure to upgrade accordingly. Other updates include added support for emphasized Tabs and various bug fixes to our existing components and hooks. Thank you to all who contributed to this release!
+
+## New features
+- Support for isEmphasized Tabs - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3124)
+
+## Updates
+- React 18 support - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3108)
+- Update dom-helpers - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3110)
+- Upgrade workflow icons - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3019)
+
+## Fixes
+- Reduce the number of rerenders from focus hooks - [@csantos1113](https://github.com/csantos1113) - [PR](https://github.com/adobe/react-spectrum/pull/3045)
+- Fix NumberParser mistakenly returning Arabic values in non-Arabic locales  - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3116)
+- Add ActionMenu to mono-package - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3144)
+- Distinguish Talkback virtual clicks from Windows FF touch event - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3136)
+- Fixes missing 'props' from API sections of useFocus and useFocusWithin docs - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3161)
+- Fix ButtonGroup visual flicker on resize check - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3160)
+- Fix React error for Tabs when every key is disabled - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3162)
```

Comment:
technically this fixed something we broke in between releases, so it shouldn't be part of the release notes. if it'd been broken in the last release, then it would make sense here